src/StripeTerminalSdk.tsx

```diff
@@ -138,6 +139,7 @@ export interface StripeTerminalSdkType {
   setSimulatedCard(cardNumber: string): Promise<{
     error?: StripeError;
   }>;
+  getOfflineStatus(): Promise<OfflinePaymentStatus>;
```

Comment:
Ooc, is there a reason this returns a `Promise` instead of just an `OfflinePaymentStatus`? The values should be immediately available from the native SDKs, so I don't think we need to make this async.

Reply:
Thank a lot for your comments~. we have also considered using the synchronous method before. Later, during the debugging process, we saw the RN document mentioning the suggestion that we use the asynchronous method to avoid possible performance problems. Please kindly correct us if we understand something wrong.https://reactnative.dev/docs/native-modules-android?android-language=kotlin

Reply:
Ok, got it. I'm fine with leaving this as async and returning a promise. 👍